server/database/models/models.ts

```diff
@@ -2,7 +2,9 @@ import { DeploymentLogModel } from "@/server/database/models/deployment-logs";
 import { HistoricalAlertModel } from "@/server/database/models/historical-alert";
 import { AdminModel } from "@/server/database/models/admin";
 import { SessionModel } from "@/server/database/models/session";
+import { AlertModel } from "@/server/database/models/alert";
 
+export const UNPROCESSED_ALERTS = AlertModel.instance;
```

Comment:
```suggestion
export const ALERTS = AlertModel.instance;
```

Let's keep these the same name as the model. The table name is [tied to the model](https://github.com/dan-schel/train-disruptions/blob/7578d70d3cf3e017868a141914c3189c04cf97ca/server/database/models/alert.ts#L26) anyway. The `Alert` model is designed to differentiate between unprocessed alerts and processed ones based on whether `processedAt` is set, not which table they're in.